docs/framework/react/guides/validation.md

```diff
@@ -453,6 +453,38 @@ const form = useForm({
 />
 ```
 
+Should you want to separate the validators from the body of the component you can do so as follows.
+
+```tsx
+const userSchema = z.object({
+  age: z.number().gte(13, 'You must be 13 to make an account'),
+})
```

Comment:
yeah, I see that now... good catch

Reply:
Doesn't the example above that show the same?

https://github.com/TanStack/form/pull/1054/files#diff-455fab23b8880cb58c7a875f5666ab15daf497b6d6fc17fba4c83466b5aafaa4R490-R499

a `formSchema` is defined before and then passed into the `useForm` `validators.onChange`.

```tsx
const formSchema = z.object({
  age: z.number().gte(13, 'You must be 13 to make an account'),
})
const form = useForm({
  validatorAdapter: zodValidator(),
  validators: {
    onChange: formSchema
  },
})
```

So to show a something new it would have to go "up" again to the "Fields"-Part?

Reply:
yes, I would agree.... It should follow the same format as above in the field validators, then separate out. I'll change it. 🤟

Reply:
what do you think of this [proposal](https://github.com/TanStack/form/pull/1054/commits/2d8bc8655d62e5d39b32ce9f7e063f93a18aeac4)?

Reply:
@harry-whorlow the problem with this method is that it's **much** slower than moving the schema out of the component entirely